README.md

```diff
@@ -68,14 +106,31 @@ conda env create -f idec.yaml
 conda activate rerunnaro
 ```
 
-Step 2: Train the PCL component (with the initial environment !!). Here, the commented methods in the NaroNet main.py file are #3, #4 and #5.
+Step 2: Train the PCL component (with the initial environment !!). The commented methods in the NaroNet main.py file are #3, #4 and #5.
+
+```python
+#1
+preprocess_images(path,params['PCL_ZscoreNormalization'],params['PCL_patch_size'])
+
+#2
+patch_contrastive_learning(path,params)    
+
+#3
+#params = architecture_search(path,params,possible_params)
+
+#4
+#run_NaroNet(path,params)
+
+#5
+#get_BioInsights(path,params)
+```
 
 ```sh
 cd your_path/NaroNet/NaroNet-main/src/main.py
 python3 main.py
 ```
 
-Step 3: PCL inference process. With the trained weights, the model generates the embeddings. Run the main.py file again as in Step 2.
+Step 3: PCL inference process. With the trained weights, the model generates the embeddings. Rerun the main.py file as in Step 2.
 
 Step 4: Run the three cells from "Column-wise processing" section in the [file](https://github.com/CarolRameder/NaroNet/blob/main/EXP3/Pipeline.ipynb). This normalizes the embedding to the scale required by the Auto Encoder(AE) component. All the embeddings from all images in the dataset are merged in one ".npy". This is the input format for the AE.
 
```

Comment:
Can you add the code snippet for the "column-wise processing" and also for "laod_embs" in step 5